content/en/users/check-in/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>404 Page not found</title>
+    <link rel="stylesheet" type="text/css" href="/scss/main.css">
+    <link rel="stylesheet" type="text/css" href="/css/error.css">
+    <meta http-equiv="refresh" content="0; url='../aai/check-in/'" />
+  </head>
+  <body class="error">
+    <p><img src="https://egi.eu/wp-content/uploads/2016/05/cropped-logo_site-1-300x300.png" alt="Logo" style="height: 100px;"></p>
```

Comment:
It would be preferable to use an included image than relying on one in the wodpress site (that will get removed soon).

Reply:
We should already have a logo in the public dir.

Reply:
I think it's the main point to be fixed before being able to merge.

Reply:
The docu landing page also links to that. Will include it locally.